.gitignore

```diff
@@ -1,47 +1,48 @@
+_book
+.DS_Store
 .env
 .envrc
-package-lock.json
+.final_nyc_output
 .git
-yarn.lock
-yarn-error.log
-build/
-dist/
-test-reports
-tmp
+.idea/
+.nyc_output
+.prettierrc.json
```

Comment:
Aside from alpha sorting, this is the only change.    Added for personal dev environment joy, nothing more.